sylvia-derive/src/message.rs

```diff
@@ -791,6 +790,21 @@ impl<'a> MsgField<'a> {
         }
     }
 
+    /// Emits struct field
+    pub fn emit_pub(&self) -> TokenStream {
```

Comment:
Let's give it some more explanatory name

Reply:
Tbh I don't see a better name here. It differs from the `emit` method only in adding the `pub` before the field.

We can rethink the naming of those methods, but this would affect every emitter in the project so let's do that outside of this PR.